### PR TITLE
update version number

### PR DESCRIPTION
**Pull request by mo271**

### CODE REVIEW

File: .github/workflows/conformance.yml [hunk 1]
```diff
@@ -20,8 +20,8 @@ permissions:
 
 env:
   CONFORMANCE_REPO_HASH: ee6008ef151489a5330cd886b422af8f6ed58881
-  LIBJXL_VERSION: 0.9.0
-  LIBJXL_ABI_VERSION: 0.9
+  LIBJXL_VERSION:  0.9.1
```
Comment (eustas) [inline .github/workflows/conformance.yml:23]: extra spaces; also, do be want to quote this?

**Review by eustas — DISMISSED** [review]

File: debian/changelog [hunk 2]
```diff
@@ -1,8 +1,14 @@
...
+  * Bump JPEG XL version to 0.9.1.
+
+ -- JPEG XL Maintainers <jpegxl@google.com>  Mon, 08 Jan 2024 16:09:42 +0100
+
 jpeg-xl (0.9.0) unstable; urgency=medium
 
   * Bump JPEG XL version to 0.9.0.
 
- -- JPEG XL Maintainers <jpegxl@google.com>  Wed, 11 Jan 2023 16:12:35 +0000
+ -- JPEG XL Maintainers <jpegxl@google.com>  Mon, 08 Jan 2024 16:09:42 +0100
```
Comment (kmilos) [inline debian/changelog:11]: 22 Dec 2023?

Comment (mo271) [reply debian/changelog]: thanks, fixed

File: .github/workflows/conformance.yml (hunk 1 above)
Comment (mo271) [reply .github/workflows/conformance.yml]: fixed.
In head the chnage to ci.sh should already make sure we are always using quotes and get the spacing right here...

**Review by eustas — APPROVED** [review]

**Review by eustas — APPROVED** [review]

**Pull request merged**